azure-communication-ui/demo-app/src/calling/java/com/azure/android/communication/ui/callingcompositedemoapp/CallLauncherActivity.kt

```diff
@@ -54,14 +56,17 @@ class CallLauncherActivity : AppCompatActivity() {
     }
 
     private lateinit var binding: ActivityCallLauncherBinding
-    private val callLauncherViewModel: CallLauncherViewModel by viewModels()
+    private var callCompositeManager: CallCompositeManager? = null
```

Comment:
Would it be easier to have `private lateinit var callCompositeManager: CallCompositeManager` ?